build/vsts_build.yaml

```diff
@@ -441,7 +441,12 @@ phases:
     demands: 
         - DotNetFramework
         - msbuild
-
+    matrix:
+      IsDesktop:
```

Comment:
Desktop.

Reply:
Leaving it this way, consistent with metadata.

Reply:
It makes no sense in this context.